The error message is wrong when the final time is almost equal to the largest value in the mesh

In the script below, we create an epidemiological model using a stop time equal to 13 and 100 vertices. 
```python
# %%
import otfmi
import os
import openturns as ot
import platform
import openturns.viewer as otv

# %%
infection_rate = 0.007
healing_rate = 0.3
input_value = [infection_rate, healing_rate]

# %%
start_time = 0.0
final_time = 13.0
number_of_vertices = 100
step = (final_time - start_time) / (number_of_vertices - 1)
mesh = ot.RegularGrid(start_time, step, number_of_vertices)
vertices = mesh.getVertices()

# %%
# Select the FMU
cwd = os.getcwd()
if platform.system() == "Linux":
    path_fmu= os.path.join(cwd, "epid-linux-x86_64.fmu")
else:
    path_fmu= os.path.join(cwd, "epid-win64.fmu")

# %%
monModele = otfmi.FMUPointToFieldFunction(
    mesh,
    path_fmu,
    inputs_fmu=["infection_rate", "healing_rate"],
    outputs_fmu=["infected"],
    start_time=start_time,
    final_time=final_time)

```
The previous script produces:
```
AssertionError: The mesh final time must be >= to FMU final time.
To set the FMU final time, use the argument final_time in
FMUPointToFieldFunction constructor.
```
This is wrong because the message is inverted. It should read "<=" instead of ">=".

The bug is here:

https://github.com/openturns/otfmi/blob/64cee3d9d34355d9fcf7e51e1db77d78c2fb19a9/otfmi/otfmi.py#L740

--- a/otfmi/otfmi.py
+++ b/otfmi/otfmi.py
@@ -730,16 +730,17 @@ class OpenTURNSFMUPointToFieldFunction(ot.OpenTURNSPythonPointToFieldFunction):
         """
         mesh = self.getOutputMesh()
         mesh_min = mesh.getVertices().getMin()[0]
+        mesh_max = mesh.getVertices().getMax()[0]
+        tol = (mesh_max - mesh_min) * 1e-6
         assert (
-            mesh_min >= self.start_time
+            mesh_min + tol >= self.start_time
         ), """The mesh start time must be >= to FMU start time.\n
             To set the FMU start time, use the argument *start_time* in
             FMUPointToFieldFunction constructor."""
 
-        mesh_max = mesh.getVertices().getMax()[0]
         assert (
-            mesh_max <= self.final_time
-        ), """The mesh final time must be >= to FMU final time.\n
+            mesh_max <= self.final_time + tol
+        ), """The mesh final time must be <= to FMU final time.\n
             To set the FMU final time, use the argument final_time in
             FMUPointToFieldFunction constructor."""
 

